src/main/java/com/orasi/bluesource/Accounts.java

```diff
@@ -55,10 +41,38 @@ public Accounts(OrasiDriver driver){
 	
 	/**Page Interactions**/
 
-	/*
+	/**
+	 * Checks if the rate provided matches the rate field on the Role page
+	 * @author David Grayson
+	 * @return <code>true</code> if the rate field from the Rates table on a Project page
+	 * matches the rate on the Rates table on the Role page, <code>false</code> if not.
+	 */
+	public boolean verifyRoleRate(String rate) {
+		return driver.findWebtable(By.xpath("//th[contains(text(), 'Rate')]/../../..")).getRowWithCellText(rate, 5) != 0;
+	}
+
+	/**
+	 * Gets the rate field in the Role Table on a project page
+	 * @author David Grayson
+	 * @param role the name of the role to get the rate for
+	 */
+	public String getRoleRateFromProjectPage(String role){
+		final int colPosition = 2;
+		try{
+			Webtable tblProjectRoles = driver.findWebtable(By.xpath("//th[contains(text(), 'Rate')]/../../.."));
+			int row = tblProjectRoles.getRowWithCellText(role);
+			return tblProjectRoles.getCell(row, colPosition).getText();
+		} catch (NoSuchElementException e){
```

Comment:
In general, I don't typically catch NoSuchElementException.  Meaning I let the test fail so that the engineer knows that the element identifier has changed/broke and needs to be fixed.  So if the role table is not found for some reason, I would rather that be the failure reason rather than the rate being null.  Make sense?  

Alternatively, If you use a syncVisible(timeout, false), it catches any of those exceptions and will return a false, and you can return that up to the test class.  But that makes things a bit more complicated as you are returning a string in this method.  

`driver.findWebtable(By.xpath("//th[contains(text(), 'Rate')]/../../..")).syncVisible(5, false)`